docs-src/cloud/api-keys/api-keys.md

```diff
@@ -0,0 +1,29 @@
+---
+id: api-keys
+title: API Keys
+sidebar_label: API Keys
+description: Temporal Cloud supports API Key Authentication, which provides you with a secure way to access Temporal Cloud programmatically.
+tags:
+  - explanation
+ssdi:
+  - API Keys is a in Public Preview for Temporal Cloud.
```

Comment:
Please change the ssdi to 

API Keys is in Public Preview release status for Temporal Cloud.